packages/editor/src/core/helpers/use-get-shadow-root.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react'
+
+export function useGetShadowRoot(containerRef: React.RefObject<HTMLElement>) {
```

Comment:
Minor: This might also simply be called `useShadowRoot`, since it's returning the shadow root itself, and not a getter for it.